### PR TITLE
Test with OCaml 4.08

### PR DESCRIPTION
**Pull request by edwintorok**

### CODE REVIEW

File: tools/travis.sh [hunk 1]
```diff
@@ -9,15 +9,15 @@ set -e
...
+set -x
 sudo apt-get update
 opam repo remove --all default
-opam repo add xs-opam file:///mnt
+opam repo add xs-opam --all-switches file:///mnt
+opam switch ${OCAML_VERSION}
 opam depext -vv -y xs-toolstack
-opam switch 4.07
-opam install -j $(getconf _NPROCESSORS_ONLN) xs-toolstack
+opam install -j \$(getconf _NPROCESSORS_ONLN) xs-toolstack
```
Comment (lindig) [inline tools/travis.sh]: The `-j $(..)` can be dropped because `opam` does this automatically.

File: tools/travis.sh [hunk 2]
```diff
@@ -9,15 +9,15 @@ set -e
...
 docker pull $IMG
-docker run --rm -iv $PWD:/mnt $IMG <<'EOF'
+docker run --rm -iv $PWD:/mnt $IMG <<EOF
 set -e
-
+set -x
 sudo apt-get update
 opam repo remove --all default
-opam repo add xs-opam file:///mnt
+opam repo add xs-opam --all-switches file:///mnt
```
Comment (lindig) [inline tools/travis.sh]: This has been merged. So this needs to be re-based.

**Comment by edwintorok** [comment]
4.08 actually failed due to ppx_tools not supporting it yet. Will close this for now.

**Comment by mseri** [comment]
As far as I understood `ppx_tools` may not be updated in favour of `ppx_tools_versioned` and `omp`.

**Pull request closed without merging**